package.json

```diff
@@ -20,6 +20,7 @@
     "handlebars-loader": "^1.0.0",
     "jquery": "^2.1.3",
     "jsdom": "^8.1.0",
+    "jsdom-global": "^2.1.1",
```

Comment:
This seemed to be necessary to get the a11y-text-wand plugin to run due to needing to access something on `window` that wasn't available without `jsdom-global`.

Reply:
Do you recall what it was that was missing?

Reply:
I'll pull it out run the test again and report back.

Reply:
Without `jsdom-global`:

```
ReferenceError: window is not defined
      at InfoPanel.initAndPosition (plugins/shared/info-panel/index.js:127:9)
      at InfoPanel.render (plugins/shared/info-panel/index.js:346:18)
      at A11yTextWand.run (plugins/a11y-text-wand/index.js:23:20)
      at Context.<anonymous> (test/a11y-wand-test.js:29:16)
```

I'm not super familiar with `jsdom`, but some brief research lead me to the `jsdom-global` and `--require jsdom-global/register` solution for resolving this error.

Reply:
Thanks for checking into that. This is not a request for you, just an outloud musing; I wonder if upgrading `jsdom` as well as setting `pretendToBeVisual` to `true` in its setup would address this too.

Reply:
A quick glance at the jsdom docs seem to indicate that might be the case. It looks like the usage of `jsdom` has changed a good bit and `jsdom.env` no longer exists and this change might require some more significant refactoring.

This same error shows up for all four of the tests this PR adds and none of the other tests run.

```
 1) Screen Reader Wand "before all" hook:
     TypeError: jsdom.env is not a function
      at Object.exports.createDom (test/mock-dom/index.js:30:11)
      at Context.<anonymous> (test/a11y-wand-test.js:12:17)
```

Reply:
I misspoke, it does try to run all the tests, but they all fail with the same error from above.

Reply:
I spent a little time trying to quickly refactor the mock-dom file, but it looks like something with how the annotations are handled might be causing an issue?

I don't think I'm familiar enough with jsdom or the way this is being handled to resolve it without a good deal more research.

Reply:
I can work on upgrading JSDOM, if you'd like. I have quite a bit of experience with it if late. I can put you on the review and we can collaborate so that all gets landed.

I've already been upgrading other dependencies yesterday, so this fits pretty well with that.

What do you think?

Reply:
Sure thing.

We can put this one on pause and rebase once the JSDOM updates are merged in.